backend/src/db/providers/pendingCoursesProvider.ts

```diff
@@ -0,0 +1,28 @@
+import UserModel from 'db/models/User';
```

Comment:
Here is the exact case when you should use relative import. Just have a look: this file positioned at the same src-level folder as UserModel (`db`).

Reply:
done